lib/galaxy/model/migrations/alembic/versions_gxy/e93c5d0b47a9_alter_column_external_user_id.py

```diff
@@ -0,0 +1,31 @@
+"""alter column CustosAuthnzToken.external_user_id to increase length
+
+Revision ID: e93c5d0b47a9
+Revises: e0561d5fc8c7
+Create Date: 2023-10-08 12:11:02.024669
+
+"""
+import sqlalchemy as sa
+
+from galaxy.model.migrations.util import alter_column
+
+# revision identifiers, used by Alembic.
+revision = "e93c5d0b47a9"
+down_revision = "e0561d5fc8c7"
+branch_labels = None
+depends_on = None
+
+table_name = "custos_authnz_token"
+column_name = "external_user_id"
+
+SHORTER_LENGTH = 64
+LONGER_LENGTH = 256
+
+
+def upgrade():
+    alter_column(table_name, column_name, existing_type=sa.String(SHORTER_LENGTH), type_=sa.String(LONGER_LENGTH))
```

Comment:
@davelopez do we need the `existing_type` argument here? From the docs (as far as I can tell) it appears it's not necessary here (even for the MySQL case, since we indicate the new type). But I don't mind keeping it if that's your preference (e.g. consistency, readability, etc.)

Reply:
Yeah, I'll remove it if is not necessary :+1:  thanks!